src/components/display/display.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Display = ({ value }) => (
+  <div>{value}</div>
+);
+
+Display.propTypes = {
+  value: PropTypes.string.isRequired,
+};
```

Comment:
requirement also says default value should be ZERO... please implement this using `defaultProps` [ [see requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/DefaultProp.png) ]